### ISSUE TITLE
Readability

### ISSUE THREAD

**Issue by fgregg** [datamade/nyc-councilmatic#78]
I'm finding the lack of color contrast and small text sizes make something hard to read. In particularl are the bill abstracts, the committee labels on the home page,

**Comment by cathydeng** [comment]
consolidating font end stuff
